docs/source/complex_numbers.rst

```diff
@@ -0,0 +1,174 @@
+.. _complex_numbers-doc:
+
+Complex Numbers
+===============
+
+Complex numbers are numbers that can be expressed in the form :math:`a + bj`, where a and b are real numbers,
+and *j* is a solution of the equation :math:`x^2 = −1`. Complex numbers frequently occur in mathematics and
+engineering, especially in signal processing. Tensors of complex dtypes provide a more natural user experience
+for users and libraries (eg. TorchAudio) that previously worked around the lack of complex tensors by using
+float tensors with shape :math:`(..., 2)` where the last dimension contained the real and imaginary values.
```

Comment:
nit: there still feels like there is some tense confusion here. Part of it is that there are plenty of libraries, TODAY, that represent complex as float tensors. So it should be present tense. Consider something like, "Tensors of complex dtypes provide a more natural user experience for working with complex numbers, compared with maintaining float tensors with shape :math:`(..., 2)` where the last dimension contains the real and imaginary values."